asapdiscovery-docking/asapdiscovery/docking/workflows/cross_docking.py

```diff
@@ -81,9 +81,11 @@ def cross_docking_workflow(inputs: CrossDockingWorkflowInputs):
     """
 
     output_dir = inputs.output_dir
```

Comment:
Since this bit appears in all the workflows, probably better to put this as a function in the base class and add a simple test for it.

Reply:
what base class are you referring to? we haven't turned these workflows into classes yet so there wouldn't be a place to put it? Oh I guess we can do that here? https://github.com/choderalab/asapdiscovery/issues/607

Reply:
Don't these workflows all insert from `PosteraDockingWorkflowInputs` which inherits from `DockingWorkflowInputsBase`? I guess I was thinking stuck in one of those.

Reply:
This is an ongoing discussion about possibly moving to a class based system, see #596. @chrisiacovella the `Inputs` just sanitise the various attributes rather than being used to run the workflow itself.